### ISSUE TITLE
More consistent display of previous results when a new search is in progress

### ISSUE THREAD

**Issue by janiemi** [spraakbanken/korp-frontend#240]
Currently, when a search is in progress, the result of the previous search is shown in the KWIC and word picture result tabs, whereas the statistics result tab is cleared. In at least Korp 5.0.10 (on which the [old Korp of the Language Bank of Finland](https://korp.csc.fi/korp-old/#?lang=en) is based), the previous KWIC result was greyed out (but you could still select words in it) and the statistics and word picture results were shown normally until the new result was ready.

I’m wondering if it would be feasible to make the behaviour of the different result tabs the same when a new search is in progress: either clear the previous result, show it as such or grey it out. Of these, I might prefer greying out.

And if a search is aborted, should Korp show the previous result? If one result is ready when the search is aborted but the rest aren’t, Korp now shows the new result that is ready (at least the first page for KWIC), and for the rest, it shows the old result (or empty for statistics), which may make the result look somewhat inconsistent.

**Comment by majsan** [comment]
Yes, lets aim to make this the same in all result tabs. I think greying out is good. It should be obvious if a result tab is the result of the current search or not. 

I think it makes sense to let the previous search become visible again if the user changes their mind. Maybe they saw something interesting. But then we would have to make sure that all the result tabs show the results from the same search.

That some show the new result and some the old should be classified as a bug!
